integration-tests/src/tests/client/sandbox.rs

```diff
@@ -109,3 +118,40 @@ fn test_patch_account() {
     let test1_after = env.query_account("test1".parse().unwrap());
     assert_eq!(test1_after.amount, 10);
 }
+
+#[test]
+#[cfg(feature = "sandbox")]
+fn test_fast_forward() {
```

Comment:
FYI, this test will be pretty much never executed. BuildKite pipeline does not include sandbox when running tests.

Reply:
I see, would there be anything extra I'd need to do for this? If not, the python tests covers the same case either way.

Reply:
We need to add `cargo test -p integration-tests --features sandbox -- tests::client::sandbox::` somewhere to `.buildkite/pipeline.yml`. Probably to ‘cargo test integration-tests*’ step. Maybe also to nightly as well; not sure if we care.

Reply:
yeah, I'll just leave them out for now then. From the pytests already covering these cases, it shouldn't really matter. We can revisit in the future